networkmanager/traffic.go

```diff
@@ -276,7 +276,10 @@ func (monitor *trafficMonitoring) deleteTrafficChain(chain, rootChain string) (e
 
 	// Store traffic data to DB
 	if traffic, ok := monitor.trafficMap[chain]; ok {
-		monitor.trafficStorage.SetTrafficMonitorData(chain, traffic.lastUpdate, traffic.currentValue)
+		if err := monitor.trafficStorage.SetTrafficMonitorData(chain,
```

Comment:
We should not return in this case: other cleanup will not be performed. Display error.

Reply:
error added